galaxy/accounts/models.py

```diff
@@ -90,29 +93,37 @@ class Meta:
     def __str__(self):
         return self.username
 
+    # FIXME: replace with django.core.urlresolvers.reverse(..)
     def get_absolute_url(self):
         return "/users/%s/" % urlquote(self.username)
 
+    # FIXME: This method looks unused
     def get_full_name(self):
         """
         Returns the first_name plus the last_name, with a space in between.
         """
         full_name = self.full_name
         return full_name.strip()
 
+    # FIXME: This method looks unused
     def get_short_name(self):
         """Returns the short name for the user."""
         return self.short_name.strip()
 
+    # FIXME: This method looks unused
     def get_num_roles(self):
         return self.roles.filter(active=True, is_valid=True).count()
 
+    _send_email = staticmethod(send_mail)  # for testing purposes
```

Comment:
Why not mocking `send_email` function or `email_user` method?

Reply:
Here we have the same problem as with `datetime.now`. `send_mail` ref is enclosed and you can not mock it. Well, technically I can do this, but you don't want to see this 😄
Ok, I will rework it in other way.